presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestMaterializedViewQueryOptimizer.java

```diff
@@ -50,52 +69,52 @@ public void testWithSimpleQuery()
     {
         String originalViewSql = format("SELECT a, b FROM %s", BASE_TABLE_1);
         String baseQuerySql = format("SELECT a, b FROM %s", BASE_TABLE_1);
-        String expectedRewrittenSql = format("SELECT a, b FROM %s", VIEW);
+        String expectedRewrittenSql = format("SELECT a, b FROM %s", VIEW_1);
 
-        assertOptimizedQuery(originalViewSql, baseQuerySql, expectedRewrittenSql);
+        assertOptimizedQuery(baseQuerySql, expectedRewrittenSql, ImmutableMap.of(BASE_TABLE_1, ImmutableMap.of(VIEW_1, originalViewSql)));
```

Comment:
nit: we can have a separate method for the cases where only one view is provided, this will avoid so many maps creation.